standalone/index.html

```diff
@@ -216,7 +216,10 @@ <h1>WebGPU Conformance Test Suite</h1>
 
     <div id="info"></div>
     <div id="resultsVis"></div>
-    <textarea id="resultsJSON" spellcheck="false"></textarea>
+
+    <p>
+      <input type=button id=copyResultsJSON value="Copy results as JSON">
```

Comment:
nit: quotes on the html attributes?

Reply:
Elsewhere in this project I have been not using quotes (src/common/templates/cts.html) but sg, let's just keep this one internally consistent for now.